benchmark/C/Savina/bndbuffer/ProdConsBenchmark.lf

```diff
@@ -231,6 +234,7 @@ reactor ManagerReactor(bufferSize:size_t(50), numProducers:size_t(40), numConsum
     =}
     
     reaction(inConsumers) -> consume {=
+        printf("inConsumers reaction is not invoked.\n");
```

Comment:
Shouldn't this be "is invoked"?

There are a bunch more of these...
Note also that benchmarks probably should not have printf() in them because it adds to the execution time.
Perhaps you want instead DEBUG_PRINT()?  This way, the overhead is negligible unless you turn on logging.

Reply:
Same as above.